ykman-gui/qml/PivCertificateInfo.qml

```diff
@@ -13,17 +13,7 @@ ColumnLayout {
 
     onVisibleChanged: visible ? load() : ''
     function load() {
-        yubiKey.pivReadCertificate(slot, function (resp) {
-            if (resp.success) {
-                certificate = resp.cert
-            } else {
-                if (resp.error) {
-                    pivError.show(resp.error)
-                } else {
-                    pivError.show('Failed to read certificate')
-                }
-            }
-        })
+        yubiKey.refreshPiv(function () {})
```

Comment:
Cool! Can we make the callback optional so we don't have to give it empty functions? Also I think we shouild call it `refreshPivData` to distinguish it more from `YubiKey.refresh()`

Reply:
Good points!